src/components/Card/Card.js

```diff
@@ -3,14 +3,14 @@ import rotten from '../../images/spoiled-tomatillo.png'
 import good from '../../images/ripe-tomatillo.png'
 import './Card.css';
 
-const Card = ({ id, posterPath, title, rating, displayDetails }) => {
+const Card = ({ id, poster_path, title, rating, displayDetails }) => {
 
     let ratingImage = rating < 6 ? rotten : good;
     
     return (
         <div>
             <h4>{title}</h4>
-            <img  onClick={() => displayDetails(id)} src={posterPath} className="poster"></img>
+            <img  onClick={() => displayDetails(id)} src={poster_path} className="poster"></img>
```

Comment:
With accessibility in mind, make sure to include an alt tag on any and all images.  This is a good thing to look out for when reviewing each other's code.